app/components/preprint-form-upload.js

```diff
@@ -10,8 +11,31 @@ export const State = Ember.Object.extend(Ember.Freezable, {
 }).create().freeze();
 
 const component = CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
+    session: Ember.inject.service(),
+    currentUser: Ember.inject.service(),
+    fileManager: Ember.inject.service(),
     state: State.START,
     uploadState: State.START,
+    selectedNode: null,
+    user: null,
+    userNodes: Ember.A([]),
+    init() {
+        this._super(...arguments);
```

Comment:
This is a case where we might be better off passing the user in to the component, rather than fetching it in an `init()` method.